docs/source/dev_guide.rst

```diff
@@ -53,6 +53,39 @@ Do the same as above using the ``Makefile``.
     $ make lint
     $ make coverage
 
+Upgrade Database
+----------------
```

Comment:
Maybe database migration should be done automatically, otherwise chances are Twitcher's schemas in the code vs the retrieved ones from the db will mismatch and just cause the app to crash. 
Expected behaviour should be that the db and its schemas are migrated up/down on boot time to match the installed Twitcher version.

Reply:
Don't know ... I thought it would be safer to run `make migrate` intentionally. Though it might not harm when run each time before `make start`.